Crawler/Analyzers/UtilAnalyzers/PunctuationAnalyzer.cs

```diff
@@ -80,5 +91,45 @@ private List<int> WordsCountBetweenPunctuations(List<Token> tokens)
 
             return wordsCount;
         }
+
+        public double CalculatePassiveVoiceSentencesPercentage(List<PosTagToken> posTagTokens)
+        {
+	        var toBeForms = ExtractWordsFromFile(config.Value.ToBeFormsFile);
+	        var passiveVoiceSentencesCount = 0;
+	        var indexOfNextSentenceEnd = posTagTokens.FindIndex(token => token.ExtendedType == ".");
```

Comment:
What is this dot on the extended type? It looks like it should be Value

Reply:
No, it's supposed to be extended type, it represents a full stop signs (?!.) but as you wrote about "VBN", I will define it as a constant as well.